taxi/forms.py

```diff
@@ -18,16 +18,21 @@ class Meta:
 
 
 class DriverCreationForm(UserCreationForm):
-    class Meta(UserCreationForm.Meta):
+    class Meta:
         model = Driver
-        fields = UserCreationForm.Meta.fields + (
-            "license_number",
-            "first_name",
-            "last_name",
-        )
+        fields = ["username", "password1", "password2", "license_number"]
 
-    def clean_license_number(self):  # this logic is optional, but possible
-        return validate_license_number(self.cleaned_data["license_number"])
+    def clean_license_number(self):
+        license_number = self.cleaned_data.get("license_number")
+        first_three = license_number[:3]
+        if not first_three.isupper():
+            raise ValidationError(
+                "First 3 characters should be uppercase letters"
+            )
+        if len(license_number[3:]) != 5 or not license_number[3:].isdigit():
+            raise ValidationError(
+                "Last 5 characters should be digits"
+            )
```

Comment:
The `clean_license_number` method in `DriverCreationForm` and `DriverLicenseUpdateForm` classes have duplicate code. You should consider reusing the `validate_license_number` function here to keep your code DRY (Don't Repeat Yourself).